package.json

```diff
@@ -167,7 +167,8 @@
     "human-crypto-keys/node-forge": "^1.3.1",
     "human-crypto-keys/crypto-key-composer/node-forge": "^1.3.1",
     "axios": "^1.7.2",
-    "rollup": "4.43.0"
+    "rollup": "4.43.0",
+    "@inquirer/core/wrap-ansi/string-width": "4.2.3"
```

Comment:
Was having error `[ERR_REQUIRE_ESM]: require() of ES Module` so I put a version that doesn't throw that error. To see the error you can try to run the dev build.